platform-modules/actors/src/main/java/org/ekstep/learning/framework/FrameworkStore.java

```diff
@@ -0,0 +1,106 @@
+package org.ekstep.learning.framework;
+
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.ekstep.cassandra.connector.util.CassandraConnector;
+import org.ekstep.cassandra.store.CassandraStore;
+import org.ekstep.common.Platform;
+import org.ekstep.common.exception.ResourceNotFoundException;
+import org.ekstep.common.exception.ResponseCode;
+import org.ekstep.common.exception.ServerException;
+import org.ekstep.learning.contentstore.ContentStoreParams;
+import org.ekstep.searchindex.util.CompositeSearchConstants;
+import org.ekstep.telemetry.logger.TelemetryManager;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * FrameworkStore handles all CRUD Operation into cassandra for Framework Hierarchy.
+ * @author Kumar Gauraw
+ */
+public class FrameworkStore extends CassandraStore {
+
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
Move it (ObjectMapper) to BaseClass and make it protected.

Reply:
done